draft-ietf-privacypass-architecture.md

```diff
@@ -282,11 +296,12 @@ See {{extensions}} for requirements on new issuance protocol variants and
 related extensions.
 
 Clients obtain the Issuer public key directly from the Origin using the process
-described in {{AUTHSCHEME}}. Clients MAY apply some form of key
+described in {{AUTHSCHEME}}. Clients MAY apply some form of
```

Comment:
Should this be elevated to a SHOULD instead of a MAY?

Reply:
I think it depends on your threat model. Maybe we just drop normative language here altogether and say "Clients can apply some form of consistency check... Failure to do so may allow Client-specific keys to impact the Client anonymity set."

Reply:
I made this change -- how's it look?

Reply:
Looks good!